book.rb

```diff
@@ -0,0 +1,9 @@
+# Represents the books class
+class Book
+  attr_accessor :title, :author
```

Comment:
- Hey, let's kindly add the rentals to the setters/getters method to make it accessible when you want to read its values.